Add: global css variables.

labiker opened the pull request:
Other content:
- Delete the color value related code in `scss` that has not been modified for more than one year after being commented.
- Replace the code about color values in `scss` and `css` files with the use of defined `css variables`.
- For components that use `iconPark` icons, delete the color value part in the `ts` file and define the color in the `scss` file instead.
- Because the visual effect of some `DefaultButton` after using `optionButtonStyles` to customize the color is the same as that of `PrimaryButton`, delete its color definition and use `PrimaryButton` instead.


**Note: The visual effect before modification and after modification is the same (checked).**

Problem: Currently, the naming of CSS variables is not standardized. Although I have improved the naming methods as much as possible through communication with `Github Copilot`, I cannot guarantee that they comply with the Owner's naming convention. Please give your opinion.

MakinoharaShoko commented:
部分颜色设置不正确，请检查

labiker commented:
> 部分颜色设置不正确，请检查

It would be helpful if you could give some hints about the areas where the unusual colors are located.

MakinoharaShoko commented:
![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/30483415/488b461d-19cc-4354-824e-fb642b283010)
![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/30483415/1e62050b-8306-4753-94fe-9e4c464fc12e)
![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/30483415/57854149-1394-4810-bbd3-a806e42ade48)

MakinoharaShoko commented:
In summary, the first step is to accurately restore the previous color scheme, and then we can consider the issue of adjustments.

MakinoharaShoko commented:
![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/30483415/94f18c9f-ba32-43b5-b833-8b8aa4f15f6c)
the color of IconPark is also not correct

MakinoharaShoko commented:
Other issues have been identified, you forgot to define --terre-primary-color, after modification most of UI is correct, but the icons still have problems

labiker commented:
Fixed.

![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/49630998/2fbb6706-15e5-4b03-80c7-c9ea6802c521)
![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/49630998/aef19531-d598-4e93-a8c4-e601563d9d4e)

MakinoharaShoko commented:
Most of the UI looks fine, only the icon for adding statements still looks a bit off

labiker commented:
Fixed.

It looks like the structure of each icon in `iconPack` is unique, no special rules can be used for accurate css coloring. At present, I have deleted all of my custom coloring, and the icon has been restored to its original state.

![image](https://github.com/MakinoharaShoko/WebGAL_Terre/assets/49630998/9879d2d6-1273-4165-8241-a8b9775ce447)
